src/templates/zh-CN/2020/methodology.html

```diff
@@ -0,0 +1,426 @@
+      </p>
+    </section>
+
+    <section>
+      <h3 id="rework-css"><a href="#rework-css" class="anchor-link">Rework CSS</a></h3>
+
+      <p>
+        <a href="https://github.com/reworkcss/css">Rework CSS</a> 是一个基于JavaScript的CSS解析器。它采用整个样式表并生成一个json编码的对象，以区分每个样式规则、选择器、指令和值。
+      </p>
+
+      <p>
+        在<a href="{{ url_for('chapter', year=year, lang=lang, chapter='css') }}">CSS</a> 章节，这个特殊用途的工具显著地提高了数据库中许多度量标准的准确性。所有外部样式表中的CSS和每个页面的内联样式块都被解析和查询，让这个分析工作成为可能。阅读<a href="https://discuss.httparchive.org/t/analyzing-stylesheets-with-a-js-based-parser/1683">这段</a> 来了解更多关于它如何同BigQuery上的HTTP Archive数据集来集成。
+      </p>
+    </section>
+
+    <section>
+      <h2 id="process"><a href="#process" class="anchor-link">分析过程</a></h2>
+
+      <p>
+        Web Almanac网络年鉴花了大约一年的时间来计划和执行，与来自网络社区的许多贡献者进行协调。本节描述我们为什么选择您在Web Almanac网络年鉴中看到的指标，以及如何查询和解释它们。
+      </p>
+    </section>
+
+    <!-- To be translated <section>
+      <h3 id="brainstorming"><a href="#brainstorming" class="anchor-link">ブレインストーミング</a></h3>
+
+      <p>
+        The 2020 Web Almanac kicked off in June 2020 as a <a href="https://twitter.com/rick_viscomi/status/1273135952848977920?s=20">request for new authors</a>. After a round of brainstorming, 22 chapters were solidified and we began <a href="https://github.com/HTTPArchive/almanac.httparchive.org/issues/876">assigning subject matter experts and peer reviewers to each chapter</a>. This process had some inherent bias because of the challenge of getting volunteers to commit to a project of this scale. Thus, many of the contributors are members of the same professional circles. One explicit goal for future editions of the Web Almanac is to encourage even more inclusion of underrepresented and heterogeneous voices as authors and peer reviewers.
+      </p>
+
+      <p>
+        We spent July 2020 pairing people with chapters and getting their input to finalize the individual metrics that will make up each chapter.
+      </p>
+    </section>
+
+    <section>
+      <h3 id="analysis"><a href="#analysis" class="anchor-link">Analysis</a></h3>
+
+      <p>
+        In July and August 2020, with the stable list of metrics and chapters, data analysts triaged the metrics for feasibility. In some cases, <a href="https://github.com/HTTPArchive/legacy.httparchive.org/tree/master/custom_metrics">custom metrics</a> needed to be created to fill gaps in our analytic capabilities.
+      </p>
+
+      <p>
+        Throughout August 2020, the HTTP Archive data pipeline crawled several million websites, gathering the metadata to be used in the Web Almanac.
+      </p>
+
+      <p>
+        The data analysts began writing queries to extract the results for each metric. In total, <strong>TBC queries</strong> were written by hand! You can browse all of the queries by chapter in the <a href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2020">sql/2020</a> directory of the project's GitHub repository.
+      </p>
+    </section>-->
```

Comment:
Can you translate this whole section and remove the `<!--` and `-->` marks when you are done?

```suggestion
    <section>
      <h3 id="brainstorming"><a href="#brainstorming" class="anchor-link">ブレインストーミング</a></h3>

      <p>
        The 2020 Web Almanac kicked off in June 2020 as a <a href="https://twitter.com/rick_viscomi/status/1273135952848977920?s=20">request for new authors</a>. After a round of brainstorming, 22 chapters were solidified and we began <a href="https://github.com/HTTPArchive/almanac.httparchive.org/issues/876">assigning subject matter experts and peer reviewers to each chapter</a>. This process had some inherent bias because of the challenge of getting volunteers to commit to a project of this scale. Thus, many of the contributors are members of the same professional circles. One explicit goal for future editions of the Web Almanac is to encourage even more inclusion of underrepresented and heterogeneous voices as authors and peer reviewers.
      </p>

      <p>
        We spent July 2020 pairing people with chapters and getting their input to finalize the individual metrics that will make up each chapter.
      </p>
    </section>

    <section>
      <h3 id="analysis"><a href="#analysis" class="anchor-link">Analysis</a></h3>

      <p>
        In July and August 2020, with the stable list of metrics and chapters, data analysts triaged the metrics for feasibility. In some cases, <a href="https://github.com/HTTPArchive/legacy.httparchive.org/tree/master/custom_metrics">custom metrics</a> needed to be created to fill gaps in our analytic capabilities.
      </p>

      <p>
        Throughout August 2020, the HTTP Archive data pipeline crawled several million websites, gathering the metadata to be used in the Web Almanac.
      </p>

      <p>
        The data analysts began writing queries to extract the results for each metric. In total, <strong>TBC queries</strong> were written by hand! You can browse all of the queries by chapter in the <a href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2020">sql/2020</a> directory of the project's GitHub repository.
      </p>
    </section>
```